test/integration/application-log-assersion/karma-reporter/index.js

```diff
@@ -0,0 +1,50 @@
+"use strict";
```

Comment:
Hi,
Yes. The karma-cucumber-reporter consumes log.suite[0] as the feature name. However, in karma-ui5, this is the file path of the test suite.
The reporter doesn't give an option to customize that, so I'll have to rewrite the reporter. Maybe I could add the reporter as a devDependency and override the function with another customized reporter but I think I'll just keep things simple just now. Any suggestions?

Reply:
The main problem I see here is that code from a different location is just copied over without a proper attribution and inclusion of copyright and license information.

Apart from that (which seems to be possible as the project is licensed under MIT) I think this shows that there are some gaps to properly use karma-ui5 with the cucumber-reporter, right?

Maybe we should rather name this new option more explicitly to support the UI5 Gherkin reporting, as I think both are tightly coupled. Then we could adopt `log.suite[0]` to include the feature name, as the testsuite path might not be that important. We could even just combine both.
What do you think?

Reply:
That would be wonderful. How about we keep the current config, so that users can still log assertions safely while we add another config to change the log.suite[0] into feature name.

Reply:
Ok, that sounds reasonable to me. So then let's not use the karma-cucumber-reporter for now and instead just check for the general reporting like the other integration tests do.

In general our integration tests should also validate the exact reported results (like tests / error messages), as this doesn't seem to be really checked right now.

But I think this should be done afterwards. For now let's just have a basic check for the number of reported tests in the console output.